band/templates/includes/carousel/carousel.html

```diff
@@ -1,6 +1,7 @@
 <div id="myCarousel" class="carousel slide" data-ride="carousel">
 
     <!-- Indicators -->
+
     <ol class="carousel-indicators">
       {% for item in carousel_items %}
```

Comment:
what if carousel_items is empty ? if we don't enter any data in back end that will throw error in front end 
maybe check the size of carousel array before rendering it